packages/commonwealth/client/scripts/state/api/threads/searchThreads.ts

```diff
@@ -92,15 +92,11 @@ const useSearchThreadsQuery = ({
         threadTitleOnly,
       }),
     {
-      getNextPageParam: (lastPage) => {
-        const nextPageNum = lastPage.page + 1;
-        if (nextPageNum <= lastPage.totalPages) {
-          return nextPageNum;
-        }
-        return undefined;
```

Comment:
I think this is only for the [search result page](https://github.com/hicommonwealth/commonwealth/blob/master/packages/commonwealth/client/scripts/views/pages/search/index.tsx), do we need to remove this?

Reply:
Ah I see. Basically we never actually use the getNextPage (You can check on master, we don't make subsequent queries in the search result page). I will work on fixing this functionality as part of this PR.

Reply:
I tried on `commonwealth.im`, and the pagination seems to work (sometimes reloading the page would trigger it if it doesnt work on scroll)

https://github.com/hicommonwealth/commonwealth/assets/51641047/6718f480-32d3-4549-a2a4-88e88cf551b3

Reply:
Yeah seems that you are correct, for whatever reason I can't find the actual logic in the code where it fetches.

Anyways it seems something is broken on the backend, becuase it is computing the count wrong. This is making me expect there to be more items after pagination, but none actually coming in. I will draft this PR while I fix the count.

Reply:
For the pagination calls, it comes from [here](https://github.com/hicommonwealth/commonwealth/blob/master/packages/commonwealth/client/scripts/views/pages/search/index.tsx#L124)